### PR TITLE
Sourcery Starbot ⭐ refactored pokulo/errno_converter

### PR DESCRIPTION
**Pull request by SourceryAI**
Thanks for starring [sourcery-ai/sourcery](/sourcery-ai/sourcery) ✨ 🌟 ✨

Here's your pull request refactoring your most popular Python repo.

If you want Sourcery to refactor all your Python repos and incoming pull requests [install our bot](/apps/sourcery-ai/installations/new).

<details>
<summary>Review changes via command line</summary>
<p>To manually merge these changes, make sure you're on the <code>main</code> branch, then run:</p>
<pre>
git fetch https://github.com/sourcery-ai-bot/errno_converter main
git merge --ff-only FETCH_HEAD
git reset HEAD^
</pre>
</details>

### CODE REVIEW

File: src/converter_lib.py [hunk 1]
```diff
@@ -69,21 +69,20 @@ def convert(cls, number=None, code=None, verbose=False):
...
+        elif number is None:
+            return str(n)
         else:
-            if code is None:
-                return c
-            elif number is None:
-                return str(n)
-            else:
-                return "{n:3d} - {c}".format(n=n, c=c)
+            return "{n:3d} - {c}".format(n=n, c=c)
```
Comment (SourceryAI) [inline src/converter_lib.py:77]: Function `Converter.convert` refactored with the following changes:

- Remove unnecessary else after guard condition ([`remove-unnecessary-else`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/remove-unnecessary-else/))

File: src/converter_lib.py [hunk 2]
```diff
@@ -69,21 +69,20 @@ def convert(cls, number=None, code=None, verbose=False):
...
 
     @classmethod
     def parse(cls):
         if len(sys.argv) < 2:
             cls.print_help()
             exit()
 
-        invalids = dict()
+        invalids = {}
```
Comment (SourceryAI) [inline src/converter_lib.py:85]: Function `Converter.parse` refactored with the following changes:

- Replace `dict()` with `{}` ([`dict-literal`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/dict-literal/))
- Replace call to format with f-string [×2] ([`use-fstring-for-formatting`](https://docs.sourcery.ai/Reference/Default-Rules/refactorings/use-fstring-for-formatting/))

**Comment by pokulo** [comment]
I will raise minor version in pyproject.toml in this MR, so I can tag and release the merge-commit immediately.

File: src/converter_lib.py (hunk 1 above)
Comment (pokulo) [reply src/converter_lib.py:77]: could be even more simple, if you had applied this rule recursively ;)

File: src/converter_lib.py (hunk 2 above)
Comment (pokulo) [reply src/converter_lib.py:85]: this is fine.

**Pull request merged**